src/Endpoints/SlovenskaPostaEndpoint.php

```diff
@@ -62,7 +62,7 @@ public function parseEvent($event): State
             'description' => null,
             'original' => [],
         ];
-        $return['date'] = Chronos::create($event['date'][0], $event['date'][1], $event['date'][2], $event['date'][3], $event['date'][4], $event['date'][5], 'Europe/Bratislava')->setTimezone('UTC');
+        $return['date'] = Chronos::createFromDate($event['date'][0], $event['date'][1], $event['date'][2], 'Europe/Bratislava')->setTime($event['date'][3], $event['date'][4], $event['date'][5])->setTimezone('UTC');
```

Comment:
:tophat:  `Europe/Bratislava` -> `TZ_BA`

Reply:
Dik, len toto nie je nas repo.

Reply:
sorry, to mi ani nenapadlo pozriet :sweat_smile: